models/database.go

```diff
@@ -371,6 +371,28 @@ var databaseSchema = [][]string{
 			PRIMARY KEY (id)
 		)`,
 	},
+
+	23: {
+		`CREATE TABLE ia_templates (
+			name VARCHAR NOT NULL,
+			version INTEGER NOT NULL,
+			summary VARCHAR NOT NULL,
+			tiers JSONB,
+			expr VARCHAR NOT NULL,
+			params JSONB,
+			"for" BIGINT,
+			severity VARCHAR NOT NULL,
+			labels TEXT,
+			annotations TEXT,
+			source VARCHAR NOT NULL,
+			yaml TEXT,
```

Comment:
Why not NOT NULL?

Reply:
Was not sure that yaml source always will be present. Will change it.

Reply:
Well, it will be for all templates stored in the database. SaaS templates will not be in the database at all, for example.

But even in that case we could store an empty string. Again, just to avoid NULLs

Reply:
Done